packages/core/content-releases/server/src/controllers/release.ts

```diff
@@ -156,43 +156,58 @@ const releaseController = {
     ctx.body = { data };
   },
 
-  /* @TODO: Migrate to new api 
   async mapEntriesToReleases(ctx: Koa.Context) {
-    const { contentTypeUid, entriesIds } = ctx.query;
+    const { contentTypeUid, documentIds, locale } = ctx.query;
 
-    if (!contentTypeUid || !entriesIds) {
+    if (!contentTypeUid || !documentIds) {
       throw new errors.ValidationError('Missing required query parameters');
     }
 
     const releaseService = getService('release', { strapi });
 
-    const releasesWithActions = await releaseService.findMany(
-      contentTypeUid,
-      entriesIds
-    );
+    const releasesWithActions = await releaseService.findMany({
+      where: {
+        releasedAt: null,
+        actions: {
+          contentType: contentTypeUid,
+          entryDocumentId: {
+            $in: documentIds,
+          },
+          locale,
+        },
```

Comment:
just in case, it's not possible to make a release to publish and unpublish the same entry right? Asking because the publishedAt value is not specified here, but I assume that's the case

Reply:
Yes, is not possible. When you try to add the same entry a second time it should throw an error